roles/openshift_facts/library/openshift_facts.py

```diff
@@ -434,10 +435,17 @@ def set_aggregate_facts(facts):
         all_hostnames.add(facts['common']['public_hostname'])
 
         if 'master' in facts:
+            # FIXME: not sure why but facts['dns']['domain'] fails
```

Comment:
We only define this role well after we have called openshift_facts on the master host a few times previously, we definitely do not want to rely on that value being set.  We also seem to currently hardcode that value to cluster.local within the openshift_master role anyway.